### ISSUE TITLE
Move visualisation of topic and keywords in student modal view

### ISSUE THREAD

**Issue by paolo-tonella** [phdfbk/phdfbk.github.io#19]
Non sono convinto che sia una buona idea visualizzare le keyword sotto il nome di ogni studente nella griglia e nella vista modale; secondo me qui e’ meglio visualizzare il topic, mettendo le keyword solo nella vista modale, magari lasciando la frase introduttiva attuale: "My research is on   “ <keywords>.

**Comment by napo** [comment]
Sulla griglia eviterei la parola "my research" perché poi diventa ripetitiva e annoia
Ho usato keyword per capire come appariva
Dal mio punto di vista possiamo eliminare le keyword
Quello che hai visto fino ad ora sono contenuti che ho inserito agendo sul foglio di calcolo online

In ogni caso la griglia studenti ha bisogno di essere arricchita altrimenti sembra l'elenco delle figurine ;)

**Comment by paolo-tonella** [comment]
Nella griglia metterei il topic (al posto delle keyword), senza alcuna frase fissa introduttiva. Nella vista modale metterei topic sotto al nome (al posto delle keyword); solo nella vista modale metterei le keyword introdotte da "My research is on “.

**Comment by paolo-tonella** [comment]
Se l'elenco delle keyword e' vuoto, la frase "My research is on " non dovrebbe comparire (in alcuni come Ivan Donadello compare; in altri come Didem Gundogdu no).

**Comment by napo** [comment]
https://github.com/phdfbk/phdfbk.github.io/commit/3dabc8432b14adda6fd5159dcbc846a23d596c65

**Comment by paolo-tonella** [comment]
Al posto delle keyword, dopo "My research is on " attualmente viene ripetuto il topic (vedi ad esempio Benjamin Bittner).

**Comment by napo** [comment]
Sorry
Avevo interpretato male 
Ora sotto il nome ci sono le keyworda
https://github.com/phdfbk/phdfbk.github.io/commit/172c292d03e4cc39e4cc45cfa5db633392633daa

Il caso "Bittner" è molto particolare perché ne ha molte 

Verifica se ti va bene

**Comment by paolo-tonella** [comment]
No: andava bene come prima, ovvero sotto il nome il topic. Semplicemente non ho trovato nessun esempio con le keyword, per vedere dove finiscono, tranne Bitter, dove pero' erano sparite.

Ricapitolando:
- topic sotto l'immagine nella griglia
- topic sotto al nome nella modale
- keywords precedute da "My research is on" nella modale

**Comment by napo** [comment]
[x] topic sotto l'immagine nella griglia
qui non sono servite modifiche
[x] topic sotto al nome nella modale
ho riportato alla situazione precedente
[x] keywords precedute da "My research is on" nella modale
ho inserito come richiesto, ma non sono convinto. Piu' che altro non so come introdurle.
Sono quasi dell'idea di levarle il campo.

Ho invece aggiornato il testo di Bittner (ho guardato la sua pagina linkedin e fatto un brutale copia/incolla da li').
Fra gli studenti quindi risulta l'unico con tutti i campi compilati.
Questo impatta notevolmente sulla finestra modale, anche perche', per come e' costruita, va a rubare spazio alla foto quando il testo e' troppo grande.
Vedi qui

![test_bittner](https://cloud.githubusercontent.com/assets/69814/17851520/a9e54500-6862-11e6-88aa-4eebb2b9c784.jpg)
